README.md

```diff
@@ -21,3 +21,8 @@ NOTE: please, make sure that the folder which contains the subproject and the pr
 ## Connect parameters and supported services types
 Each connect can support several service types and each one can have its own parameters to configure.
```

Comment:
Each **connect extension** can support several service types and each one can have its own parameters to configure.